packages/web/components/complex/paginated-table.tsx

```diff
@@ -156,31 +159,31 @@ export const PaginatedTable = ({
         {virtualRows.map((virtualRow) => {
           const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
           return (
-            <tr
-              key={row.id}
-              className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
-              onClick={() => {
-                router.push(getPoolLink(row.original.queryPool));
-              }}
-            >
-              {row.getVisibleCells().map((cell) => {
-                return (
-                  <td key={cell.id}>
-                    <Link
-                      href={getPoolLink(row.original.queryPool)}
-                      key={virtualRow.index}
-                      passHref
-                      onClick={(e) => e.stopPropagation()}
-                    >
-                      {flexRender(
-                        cell.column.columnDef.cell,
-                        cell.getContext()
-                      )}
-                    </Link>
-                  </td>
-                );
-              })}
-            </tr>
+<tr
+  key={row.id}
+  className="transition-colors focus-within:bg-osmoverse-700 focus-within:outline-none hover:cursor-pointer hover:bg-osmoverse-800"
+>
+  {row.getVisibleCells().map((cell) => {
+    return (
+      <td
+        key={cell.id}
+        onClick={() => {
+          if (row.original.queryPool.type !== "transmuter") {
+            router.push(getPoolLink(row.original.queryPool));
+          }
+        }}
+      >
+        {row.original.queryPool.type === "transmuter" ? (
+          <a href={getPoolLink(row.original.queryPool)} target="_blank" rel="noopener noreferrer">
```

Comment:
Why did you replace `Link` component with `a` tag?

Reply:
The Link component was replaced with an a tag because the Link component from next/link does not support the target="_blank" attribute. This attribute is used to open the link in a new tab. The a tag was used instead to allow for this functionality.